providers/ms365/resources/ms365.lr

```diff
@@ -851,6 +851,34 @@ microsoft.policies {
   permissionGrantPolicies() []dict
   // Consent policy settings
   consentPolicySettings() dict
+  // Authentication methods policy
+  authenticationMethodsPolicy() microsoft.policies.authenticationMethodsPolicy
+}
+
+// The tenant-wide policy that controls which authentication methods are allowed.
+private microsoft.policies.authenticationMethodsPolicy @defaults("id displayName") {
+  // Policy ID
+  id string
+  // Policy display name.
+  displayName string
+  // Policy description.
+  description string
+  // The date and time the policy was last modified.
+  lastModifiedDateTime time
+  // The version of the policy.
+  policyVersion string
+  // Configurations for specific authentication methods.
+  authenticationMethodConfigurations []microsoft.policies.authenticationMethodConfiguration
+}
+
+// Configuration for a specific authentication method.
+private microsoft.policies.authenticationMethodConfiguration @defaults("id state") {
```

Comment:
The id is not anything the user cares about so probably shouldn't be part of the defaults